cpp/oneapi/dal/test/engine/catch.hpp

```diff
@@ -27,4 +27,6 @@
 // To make USM work, we disable signal handling in Catch2.
 #define CATCH_CONFIG_NO_POSIX_SIGNALS
 
+#define CATCH_CONFIG_DISABLE_EXCEPTIONS
```

Comment:
Why this is needed?

Reply:
If Catch2 does not catch exceptions from test cases, it's easier to debug via GDB.